index.html

```diff
@@ -32,19 +39,14 @@
                         {{/if}}
                     <span class="email-catalog_email_body">{{body}}</span>
                 </div>
+            {{else}}
+                <i>No emails here!</i>
```

Comment:
Use <em> tag instead of <i> to indicate emphasis. According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em ,  we should only use <i> when referring to stuff like movies